assets/queries/crossplane/aws/cloudfront_without_minimum_protocol_tls_1.2/query.rego

```diff
@@ -0,0 +1,135 @@
+package Cx
+
+import data.generic.common as common_lib
+
+CxPolicy[result] {
```

Comment:
IMHO, I think we should also verify if `resource.spec.forProvider.distributionConfig.enabled` is not set to false. If set to false, the recommended configuration will be pointless. The same applies for the other policies